docs/en/design/visualization.md

```diff
@@ -1,3 +1,76 @@
 # Visualization
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/design/visualization.html).
+## 1 Overall Design
+
+Visualization provides an intuitive explanation of the training and testing process of the deep learning model. In OpenMMLab, we expect the visualization function to meet the following requirements:
+
+- Provides rich out-of-the-box features that can meet most computer vision visualization tasks.
+- High scalability through easy extension and customization, as visualization functions are usually highly diversified.
+- Able to visualize at any point in the training and testing process.
+- A group of unified APIs to all OpenMMLab libraries, which is convenient for users to understand and maintain.
+
+Based on the above requirements, we introduced the `Visualizer` and various `VisBackend` such as `LocalVisBackend`, `WandbVisBackend`, and `TensorboardVisBackend` in OpenMMLab 2.0. The visualization functions work not only on the image data but also things like configurations, scalars, and model diagrams.
+
+- For convenience, the APIs provided by the `Visualizer` implement the drawing and storage functions. `VisBackend`, as an internal property of `Visualizer`, will be called by `Visualizer` when needed to store data to different backends.
+- Considering that you may want to store multiple backends after drawing, `Visualizer` can be configured with various VisBackends. When the user calls the storage API of the `Visualizer`, it will traverse and call all the specified APIs of `VisBackend` internally.
+
+The UML relationship diagram of the two is as follows.
+
+<div align="center">
+ <img src="https://user-images.githubusercontent.com/17425982/163327736-f7cb3b16-ef07-46bc-982a-3cc7495e6c82.png" >
+</div>
+
+## 2 Visualizer
+
+`Visualizer` exposed all APIs to the outside. These APIs can be divided into three categories.
+
+1. Drawing APIs
+
+- [draw_bboxes](mmengine.visualization.Visualizer.draw_bboxes) draws a single or multiple bounding boxes
+- [draw_points](mmengine.visualization.Visualizer.draw_points) draws a single or multiple points
+- [draw_texts](mmengine.visualization.Visualizer.draw_texts) draws a single or multiple text boxes
+- [draw_lines](mmengine.visualization.Visualizer.lines) draws a single or multiple line segments
+- [draw_circles](mmengine.visualization.Visualizer.draw_circles) draws a single or multiple circles
+- [draw_polygons](mmengine.visualization.Visualizer.draw_polygons) draws a single or multiple polygons
+- [draw_binary_masks](mmengine.visualization.Visualizer.draw_binary_mask) draws single or multiple binary masks
+- [draw_featmap](mmengine.visualization.Visualizer.draw_featmap) draws feature map (**static method**)
+
+The above APIs can be called in a chain except for `draw_featmap` because the image size may change after this method is called. To avoid confusion, `draw_featmap` is set to a static method.
```

Comment:
```suggestion
The above APIs can be called in a chain except for `draw_featmap` because the image size may change after this method is called. To avoid confusion, `draw_featmap` is a static method.
```